test/test-functions

```diff
@@ -209,6 +210,7 @@ BASICTOOLS=(
     sfdisk
     sh
     sleep
+    sort
```

Comment:
Is this used?

Reply:
This is already part of the "$DEBUGTOOLS" array below.